### ISSUE TITLE
Two comments standing side by side in a selector are read as an inline comment by `isStandardSyntaxSelector`

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#164]
Found by an adversarial review of the branch that repairs [#161](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/161), and unrelated to it: this needs no preprocessor syntax and reproduces on `main` as it stands.

### What minimal example or steps are needed to reproduce the bug?

Two comments standing side by side in a selector:

```css
a:not(  b  )/*one*//*two*/c {}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/selector-pseudo-class-parentheses-space-inside": "never"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### Which Stylelint-related dependencies are you using?

```json
{
  "@stylistic/stylelint-plugin": "5.3.0",
  "stylelint": "17.14.1"
}
```

### What did you expect to happen?

```
1:7   Unexpected whitespace after "("
1:11  Unexpected whitespace before ")"
```

which is what the same selector draws the moment a space is put between the two comments: `a:not(  b  )/*one*/ /*two*/c {}`.

### What actually happened?

Nothing is reported, and nothing is fixed. The rule node is passed over, and so is every other rule of this plugin that asks whether the selector is standard syntax.

### Do you have a proposal to fix the bug?

`isStandardSyntaxSelector` ends with

```js
//  SCSS and Less comments
if (selector.includes(`//`)) return false
```

and the closing `*/` of one comment followed by the opening `/*` of the next spells `//` between them. So `/*one*//*two*/` reads as an inline comment, and with it the whole selector reads as a preprocessor construct.

The double slash of an inline comment is one that opens a comment, and a search that already knows where the block comments are cannot mistake `*//*` for one. `lib/utils/findInlineCommentSpans` answers exactly that question for a value; the same reading applied here, or simply blanking the block comments before the test, closes it.

The rules that fall silent are the ones the util speaks for: `no-extra-semicolons`, `selector-attribute-brackets-space-inside`, `selector-descendant-combinator-no-non-space`, `selector-list-comma-newline-after`, `selector-pseudo-class-case`, `selector-pseudo-class-parentheses-space-inside`, `selector-pseudo-element-case` and `string-quotes` — the same list as [#151](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/151), which is the same util read too plainly in another way. The two want fixing together.
